examples/dt/uni08theta/sriov_amd-sev/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-amd-sev
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - libvirt
+    - ovn
+    - neutron-metadata
```

Comment:
do we need neutron-sriov here?
I have this in a (non AMD) dpdk+sriov deployment:
~~~
[ciuser@tigon31 ~]$ oc get OpenStackDataPlaneNodeSet openstack-edpm-ovsdpdksriov -o json | jq .spec.services
[
  "repo-setup",
  "repo-setup",
  "download-cache",
  "bootstrap",
  "reboot-os",
  "configure-ovs-dpdk",
  "configure-network",
  "validate-network",
  "install-os",
  "configure-os",
  "run-os",
  "ovn",
  "neutron-metadata",
  "neutron-sriov",
  "libvirt",
  "nova-custom-ovsdpdksriov",  <- for amd this must be changed by sriov-amd as it is actually done below
  "telemetry"
]
~~~

Reply:
Does [1] have everything needed for Neutron?  If so then it is covered by `sriov-amd` if not it can be updated to include what is necessary.

[1] https://github.com/openstack-k8s-operators/architecture/pull/86/files#diff-16f7edb9ed6113137ae3d29f55aec25670de0c1ca4adf2ab1702350adfabd49b

Reply:
having neutron-sriov depend on are we testing with the sriov-nic agent or with hardware offloaded ovs.

in this case i belive yes we should have have ovs-dpdk tested via ovn and also have the sriov nic agent to test standalone sriov

you cannot use ovs-dpdk via ovn and sriov via ovn on the same host